content/case-studies/snowflake.md

```diff
@@ -9,6 +9,13 @@ customer_name: Snowflake
 customer_logo: /logos/customers/snowflake-logo.svg
 customer_url: https://www.snowflake.com/
 
+quote_block:
+  quote: |
+      There’s no question that, if we had gone down the traditional infrastructure automation route, it would have taken us a week or a week and a half to do one deployment of the cluster, end to end. Plus, that deployment wouldn’t be repeatable, testable or scalable. It would have been a mad scramble to meet our deadline. When we demonstrated to people that what used to take a week and a half now, with Pulumi, took under a day, they were shocked.
```

Comment:
```suggestion
      There’s no question that, if we had gone down the traditional infrastructure automation route, it would have taken us a week or a week and a half to do one deployment of the cluster [...] When we demonstrated to people that what used to take a week and a half now, with Pulumi, took under a day, they were shocked.
```